### PR TITLE
completed testing environment setup

### PR DESCRIPTION
**Pull request by MBellinghausen**
Environment set up with a dummy test in place that creates a generated snapshot of the render.

### CODE REVIEW

**Review by bluechocolate2019 — CHANGES_REQUESTED** [review]
Questions:
Do we need the commented out sections in App.test.js, or should we delete?

File: testing/src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    //   <App page="http://www.facebook.com">Facebook</App>,
+    // );
+    // let tree = component.toJSON();
+    // expect(tree).toMatchSnapshot();
+    // let getByText = renderer(<App />);
+    // expect(getByText("Hello World")).toMatchInlineSnapshot(`
+    //   <h1>
+    //     Hello world
+    //   </h1>
+    // `);
```
Comment (nsfoster87) [inline testing/src/App.test.js:20]: I'm just wondering about the commented out code...I'm assuming it was an attempt at writing an initial test for App component that you didn't want to delete just yet?

Comment (nsfoster87) [reply testing/src/App.test.js:20]: Or is it that we took out the Hello World message from the App component?

File: testing/src/__snapshots__/App.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          Comfortable and practical.
+        </li>
+        <li>
+          They are very dark. But that's good because I'm in very sunny spots
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+`;
```
Comment (nsfoster87) [inline testing/src/__snapshots__/App.test.js.snap:27]: We probably don't want these test outputs getting checked into git. Should this snapshots folder get added to .gitignore?

**Review by nsfoster87 — COMMENTED** [review]
The last file is a little confusing to me and looks like output from a test and I'm assuming we don't want that or any other test output getting checked into git.

**Review by bluechocolate2019 — APPROVED** [review]

**Pull request merged**